packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1788,6 +1788,56 @@ describe('TransactionController', () => {
 
       expect(delayMessengerMock.call).not.toHaveBeenCalled();
     });
+
+    it('catches error without code property in error object', async () => {
+      const mockedErrorMessage = 'mocked error';
+      (
+        delayMessengerMock.call as jest.MockedFunction<any>
+      ).mockImplementationOnce(() => {
+        // eslint-disable-next-line @typescript-eslint/no-throw-literal
+        throw { message: mockedErrorMessage };
+      });
+      const consoleSpy = jest.spyOn(console, 'error').mockImplementation();
+      const controller = newController({
+        options: {
+          disableHistory: true,
+        },
+      });
+
+      controller.state.transactions.push({
+        ...txMeta,
+        txParams: { data: '0x', from: ACCOUNT_MOCK },
+      } as any);
```

Comment:
```suggestion
      });
```

Reply:
Thanks for your suggestions I moved the `txParams` to the first object for this line, but I think all the `as any` was removed here https://github.com/MetaMask/core/pull/1867/commits/0f55018f0f69b1983cd25e7d23c8ea3f56005bef , thanks again :pray: .